release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1909,6 +1892,24 @@ In this update, `systemd` service only sets the default RPS mask for virtual int
+
+* With this update, the `jsonData` property has been deprecated and replaced with `data` in the `ping` source. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2084438[*BZ#2084438*])
+
+* Previously, topology with more than 100 nodes was failing or lagging due to size. With this update, users are alerted to the lagging issue and offered the option to use `Search` to find a specific resource or continue with topology eliminating any lag. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2060329[*BZ#2060329*])
+
+* Previously, The `8080` port was stalled on the service port list and random ports are deleted on a service with multiple ports. With this update, a condition in the `createService` function is specified to identify the current route `tartgetPort` and replace that prot from the service ports whenever a new `targetPort` is added in the edit application form. Now, when port 8080 is replaced as route port, it is removed from the service ports list and the new route ports list and the new route port gets added to the list. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2077943[*BZ#2077943*])
+
+* Previously, the git detection used to manage instance APIs to get repository information did not work for repositories from self hosted GitHub and Bitbucket. With this update, detection for self hosted GitHub and Bitbucket instance repositories works. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2038244[*BZ#2038244*])
+
+* Previously, `apiVersion` was not passed in the correct format to `Resource` dropdown in `EventSource` creation form. This caused `InContext` to not be selected under `EventSource` creation, which excluded it from the `Resource` dropdown menu. With this update, the `Resource` dropdown menu includes `Resource` from `InContext`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2070020[*BZ#2070020*])
+
+* Previously, the *pipeline metrics* page displayed all API calls for the metrics query and are failing with a `404` error. With this update, the `prometheus-tenancy` API to get the metrics data for the pipeline. Now, the pipeline metrics page displays all data and graphs to the non-admin user with at least view access to the namespace. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2041769[*BZ#2041769*])
+
+* Previously, you could access a quick search and add modal with the Crtl+space keyboard shortcut, but you could not close them by the same keyboard shortcut. With this update, you can now close the quick search and add modal using Ctrl+space keyboard shortcut. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2093586[*BZ#2093586*])
```

Comment:
You could update "close them by the same keyboard shortcut" to "close them by using the same keyboard shortcut".